koku/providers/gcp/provider.py

```diff
@@ -40,6 +40,13 @@ def cost_usage_source_is_reachable(self, credential_name, data_source):
             key = 'billing_source.bucket'
             raise serializers.ValidationError(error_obj(key, e.message))
 
+        report_prefix = data_source.get('report_prefix', '')
```

Comment:
https://github.com/project-koku/koku/blob/master/koku/api/provider/serializers.py#L159 might be a better place to put this check. I think `cost_usage_source_is_reachable` should only check whether or not GCP bucket is reachable.